### PR TITLE
@ #2 | should add kubernetes-stack-cookbook

### PR DESCRIPTION
**Pull request by ducng0610**
connect #2

### CODE REVIEW

**Comment by hoatle** [comment]
@duc4nh see my comment here https://github.com/acme101/dev-setup/issues/2#issuecomment-322971790

**Review by hoatle — CHANGES_REQUESTED** [review]
need to update with the updated json config

**Comment by ducng0610** [comment]
json config updated

File: chef/main-cookbooks/acme/recipes/k8s.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      version node['kubernetes-stack']['kubernetes']['version']
+    end
+end
+
+if node['kubernetes-stack']['gcloud']['enabled']
+    gcloud 'install gcloud' do
+      version node['kubernetes-stack']['gcloud']['version']
+    end
+
+    # fix gsutil temporarily
```
Comment (hoatle) [inline chef/main-cookbooks/acme/recipes/k8s.rb]: this is not required anymore, fixed by k8s-stack-cookbook

File: chef/main-cookbooks/acme/recipes/k8s.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#
```
Comment (hoatle) [inline chef/main-cookbooks/acme/recipes/k8s.rb:1]: please add support for "install" or "remove" action, refer here: https://github.com/iorad/dev-setup/pull/18

**Review by hoatle — CHANGES_REQUESTED** [review]

File: chef/main-cookbooks/acme/recipes/k8s.rb (hunk 2 above)
Comment (ducng0610) [reply chef/main-cookbooks/acme/recipes/k8s.rb:1]: updated

File: chef/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+
+  dev:
+    image: chef/chefdk
```
Comment (hoatle) [inline chef/docker-compose.yml]: use teracy/chefdk image instead

**Comment by hoatle** [comment]
@duc4nh please update to use teracy/chefdk image instead of chef/chefdk

**Comment by ducng0610** [comment]
updated to use teracy/chefdk

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**